bottomsheet/src/main/java/com/flipboard/bottomsheet/BottomSheetLayout.java

```diff
@@ -716,7 +716,6 @@ public void onAnimationEnd(Animator animation) {
                     // Remove sheet specific properties
                     viewTransformer = null;
                     onSheetDismissedListeners.clear();
-                    onSheetStateChangeListeners.clear();
```

Comment:
I think this makes sense. We'll have to update the documentation to make it clear that callers need to remove their listeners. We should also mention that in the change log.